### PR TITLE
fix: added aws-sts-jdk to solve irsa assume role issue

### PR DESCRIPTION
**Pull request by githubjianli**
Fixes #

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [3.5.6] - 2024-06-13
```
Comment (abhimanyugupta07) [inline CHANGELOG.md:7]:
```suggestion
## [3.5.6] - 2024-06-14
```

**Review by patduin — APPROVED** [review]

File: pom.xml [hunk 2]
```diff
@@ -35,6 +35,7 @@
   </scm>
 
   <properties>
+    <aws-java-sdk.version>1.12.276</aws-java-sdk.version>
```
Comment (abhimanyugupta07) [inline pom.xml:38]: lets use this variable here as well - https://github.com/ExpediaGroup/beekeeper/blob/main/beekeeper-cleanup/pom.xml#L14

Comment (abhimanyugupta07) [reply pom.xml:38]: use here - https://github.com/ExpediaGroup/beekeeper/blob/main/beekeeper-cleanup/pom.xml#L86

remove from here - https://github.com/ExpediaGroup/beekeeper/blob/main/beekeeper-cleanup/pom.xml#L14

Comment (githubjianli) [reply pom.xml:38]: updated, also change version to `1.12.311` to match it with previously version

**Review by DhrubajyotiSadhu — APPROVED** [review]

**Review by abhimanyugupta07 — APPROVED** [review]

**Pull request merged**